### DISCUSSION TITLE
Ability to choose Font on Script Execution log

### DISCUSSION THREAD

**Discussion by RezaSeedin** [burkybang/SuiteAdvanced-Feedback#93] [Ideas]
Hi @burkybang,

Can we have the option to choose the font in the script execution log?

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/18523110/aa4930c7-cec4-4489-b60e-0e8a7a497b38)

Font without SA enabled
![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/18523110/3fdaa109-e0b7-4f5e-85de-e7ba8aee62f2)

**Comment by burkybang** [comment]
Just the font family? The color, size, style too? How would this feature help you?

**Comment by RezaSeedin** [comment]
HI @burkybang ,

The option to choose a few popular fonts, Font sizes also would help.
Maybe an option to colour Debug vs Audit(Brown) Vs Error(RED) 

Basically, help customise the look and feel
